xdsl/transforms/experimental/ConvertStencilToLLMLIR.py

```diff
@@ -119,6 +119,49 @@ def match_and_rewrite(self, op: ApplyOp, rewriter: PatternRewriter, /):
         rewriter.replace_matched_op([zero, one, *upperBounds, p])
 
 
+class AccessOpToMemref(RewritePattern):
+
+    @op_type_rewrite_pattern
+    def match_and_rewrite(self, op: AccessOp, rewriter: PatternRewriter, /):
+        if not isinstance(op.temp.owner, memref.Cast):
+            warn(
+                "stencil.load should have been lowered before lowering related "
+                "stencil.access")
+            return
+
+        if (block := op.parent_block()) is None:
```

Comment:
Can you add a dosctringon this condition?

Reply:
Answered on webmiche's comment of the same line